blog/utils/handlePosts.ts

```diff
@@ -96,10 +115,11 @@ export const filterPosts = (
 export default function handlePosts(
   posts: BlogPost[],
   sortBy: SortBy,
-  slug?: string,
+  slug?: string | string[],
```

Comment:
missing JSDoc

 * @param slug Category slug or an array of slugs to be filtered